tests/by-util/test_install.rs

```diff
@@ -1767,3 +1775,34 @@ fn test_install_from_stdin() {
     assert!(at.file_exists(target));
     assert_eq!(at.read(target), test_string);
 }
+
+#[test]
+fn test_install_symlink_dest() {
```

Comment:
Is this test supposed to pass without the changes made to `install.rs`?

Reply:
Good question.
Tbh, I haven't been able to write a simple bash test case for this.
I did the implementation using the gnu test. Dunno why :(